.drone.star

```diff
@@ -397,7 +397,11 @@ def binary(ctx, name):
           'files': [
             'dist/release/*',
           ],
-          'title': ctx.build.ref.replace("refs/tags/", ""),
+          'checksum': [
```

Comment:
You should drop that, checksum files are already part of the Makefile, and they should stay within the Makefile as they are also uploaded to our mirrors, not only GitHub releases.

Reply:
ok, didn't know that. will drop it.

Reply:
If I can add a private GPG key to Drone we could also automatically generate `.asc` files to have something more trustable than checksum failes.